### DISCUSSION TITLE
define Conductivity in MEEP

### DISCUSSION THREAD

**Discussion by Hansji93** [NanoComp/meep#2125] [Q&A]
Hello Together,
could any of you guys help me to define the conductivity of a material in Meep. I would like to carryout Simulation for a cylinder form Beam with 3 different  Material (Pd, Cu, Ag). How do i define the Conductivity (conductivity of the Beam is known) in order to simulate the S Parameter of the Beam. Thank you.

**Comment by oskooi** [comment]
See the section [Conductivity and Complex ε](https://meep.readthedocs.io/en/latest/Materials/#conductivity-and-complex) in the user manual. The last paragraph describes how to convert from SI to Meep units.

**Reply by Hansji93** [reply]
Thank you for your quick reply. But the paragraph describes the conductivity for a medium. How do i suppose to give the sigma _D for a Beam material. E.g material=mp.metal(D_conductivity= converted value)) does this work in Meep??? Since the real relative permittivity of metal/conductor is infinity, D_conductivity is zero right?
